src/js/view/controls/components/menu/menu.js

```diff
@@ -490,7 +535,8 @@ const addGlobalMenuKeyListener = (settingsMenu) => {
                 break;
             case 'Up':
             case 'Down':
-                openMenu(key);
+            case 'Enter':
+                menu().open(evt);
```

Comment:
is there a chance for this function to not return what is expected? potential breakage here if so